LinkServer/Program.cs

```diff
@@ -19,6 +18,14 @@
 
 var app = builder.Build();
 
+//logging incoming packet,logging successfull and unsucessfull authentication 
```

Comment:
Is this chunk of code inside our Program.cs essential to the logger, or was it just being used for testing? It looks like it is logging packets and authentication even though it is placed inside of the server's startup sequence.